examples/greet-and-react/index.js

```diff
@@ -7,10 +7,9 @@ const passport = require('passport');
 const SlackStrategy = require('@aoberoi/passport-slack').default.Strategy;
 const http = require('http');
 const express = require('express');
-const bodyParser = require('body-parser');
 
-// *** Initialize event adapter using verification token from environment variables ***
-const slackEvents = slackEventsApi.createSlackEventAdapter(process.env.SLACK_VERIFICATION_TOKEN, {
+// *** Initialize event adapter using signing secret from environment variables ***
+const slackEvents = slackEventsApi.createSlackEventAdapter(process.env.SIGNING_SECRET, {
```

Comment:
we should also update the `package.json` of this example to reference the yet-to-be-published v2.0.0 so that this actually runs.